homeassistant/components/screenlogic/binary_sensor.py

```diff
@@ -85,82 +93,104 @@ class ScreenLogicPushBinarySensorDescription(
         data_root=(DEVICE.INTELLICHEM, GROUP.ALARM),
         key=VALUE.FLOW_ALARM,
         device_class=BinarySensorDeviceClass.PROBLEM,
+        translation_key="flow_alarm",
     ),
     ScreenLogicPushBinarySensorDescription(
         subscription_code=CODE.CHEMISTRY_CHANGED,
         data_root=(DEVICE.INTELLICHEM, GROUP.ALARM),
         key=VALUE.ORP_HIGH_ALARM,
         device_class=BinarySensorDeviceClass.PROBLEM,
+        translation_key="chem_high_alarm",
+        translation_placeholders={"chem": "ORP"},
     ),
     ScreenLogicPushBinarySensorDescription(
         subscription_code=CODE.CHEMISTRY_CHANGED,
         data_root=(DEVICE.INTELLICHEM, GROUP.ALARM),
         key=VALUE.ORP_LOW_ALARM,
         device_class=BinarySensorDeviceClass.PROBLEM,
+        translation_key="chem_low_alarm",
+        translation_placeholders={"chem": "ORP"},
     ),
     ScreenLogicPushBinarySensorDescription(
         subscription_code=CODE.CHEMISTRY_CHANGED,
         data_root=(DEVICE.INTELLICHEM, GROUP.ALARM),
         key=VALUE.ORP_SUPPLY_ALARM,
         device_class=BinarySensorDeviceClass.PROBLEM,
+        translation_key="chem_supply_alarm",
+        translation_placeholders={"chem": "ORP"},
     ),
     ScreenLogicPushBinarySensorDescription(
         subscription_code=CODE.CHEMISTRY_CHANGED,
         data_root=(DEVICE.INTELLICHEM, GROUP.ALARM),
         key=VALUE.PH_HIGH_ALARM,
         device_class=BinarySensorDeviceClass.PROBLEM,
+        translation_key="chem_high_alarm",
+        translation_placeholders={"chem": "pH"},
     ),
     ScreenLogicPushBinarySensorDescription(
         subscription_code=CODE.CHEMISTRY_CHANGED,
         data_root=(DEVICE.INTELLICHEM, GROUP.ALARM),
         key=VALUE.PH_LOW_ALARM,
         device_class=BinarySensorDeviceClass.PROBLEM,
+        translation_key="chem_low_alarm",
+        translation_placeholders={"chem": "pH"},
     ),
     ScreenLogicPushBinarySensorDescription(
         subscription_code=CODE.CHEMISTRY_CHANGED,
         data_root=(DEVICE.INTELLICHEM, GROUP.ALARM),
         key=VALUE.PH_SUPPLY_ALARM,
         device_class=BinarySensorDeviceClass.PROBLEM,
+        translation_key="chem_supply_alarm",
+        translation_placeholders={"chem": "pH"},
     ),
     ScreenLogicPushBinarySensorDescription(
         subscription_code=CODE.CHEMISTRY_CHANGED,
         data_root=(DEVICE.INTELLICHEM, GROUP.ALARM),
         key=VALUE.PROBE_FAULT_ALARM,
         device_class=BinarySensorDeviceClass.PROBLEM,
+        translation_key="probe_fault_alarm",
     ),
     ScreenLogicPushBinarySensorDescription(
         subscription_code=CODE.CHEMISTRY_CHANGED,
         data_root=(DEVICE.INTELLICHEM, GROUP.ALERT),
         key=VALUE.ORP_LIMIT,
+        translation_key="chem_limit",
+        translation_placeholders={"chem": "ORP"},
```

Comment:
What's ORP? Should we write it out? Does pH have different translations for other languages? Maybe it would make sense to keep them as part of the translation string

Reply:
ORP stands for Oxidation Reduction Potential.  Neither appear to be translated for other languages. Looking at the [Pentair Europe site](https://www.pentairpooleurope.com/products/automation/intellipool) and cycling through the available languages, neither ORP or pH are ever referred to by their full name, or translated.

Reply:
@joostlek After doing some more digging, it appears that while the abbreviation "ORP" is not translated, the measurement of Oxidation-Reduction Potential is also known by the term "redox" in certain regions of the world, and that term can be translated/spelled differently in different languages.
While "ORP" does appear to be used and valid globally, would it be desirable to support translation of that term (and possibly "pH"?) to allow for the name to be _changed_ in the translation to something more common in that language?

Reply:
Will leave as-is for now as the term seems to be globally understood in relation to pool chemistry.